driver-core/src/test/java/com/datastax/driver/core/EventDebouncerTest.java

```diff
@@ -65,6 +67,33 @@ public void should_deliver_single_event() throws InterruptedException {
     }
 
     @Test(groups = "unit")
+    public void should_log_and_drop_events_on_overflow() throws InterruptedException {
```

Comment:
Oh really? I thought naively it would take quite  a while to browse 10000 events, but it's probably better to avoid that if we can :+1: